index.html

```diff
@@ -217,8 +217,6 @@ <h1 id="ovningar">Övningar</h1>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
     <script src="vendor/highlight.pack.js"></script>
     <script src="vendor/bootstrap.min.js"></script>
-    <script src="vendor/smooth-scroll.js"></script>
```

Comment:
The removal of the smooth scrolling functionality really isn't an issue to me. It may stay, or it may go. As i mentioned earlier, the work that Broberg wanted us to do was to:
- expand on Helmertz' work (#3) which aimed to make the import of exercises more automatized 
- make the site easier to maintain; the removal of third party libraries being part of this.
We all got our ways of doing things, and we can argue which is better. Here, I've tried to stay as close to the style and spirit from Brook and Helmertz while doing the adaptions Broberg asked for.